src/voprf.rs

```diff
@@ -168,13 +169,13 @@ impl<G: Group, H: BlockInput + Digest> NonVerifiableClient<G, H> {
     pub fn finalize(
         &self,
         evaluation_element: EvaluationElement<G, H>,
-        metadata: &Metadata,
+        metadata: Option<&[u8]>,
```

Comment:
I was thinking that giving it a descriptive name would be a slightly better interface than simply having an Option here (since a Some(...) or None is not super descriptive). But I don't feel super strongly about this.

Reply:
I don't have a preference myself, was just avoiding a lifetime on `Metadata`. I didn't find any guidance in https://rust-lang.github.io/api-guidelines/ either.
My assumption is that the parameter name should be enough.